tests/data/power_flow/generic_branch/params.json

```diff
@@ -0,0 +1,5 @@
+{
+  "calculation_method": ["newton_raphson"],
+  "rtol": 1e-05,
```

Comment:
this is quite large for a relative tolerance. E.g. the difference in the `u` of the two nodes in the `sym_output.json` is smaller (relatively), meaning the actual output could also have `u[1] > u[0]`, which would imply a sign inversion. I'm OK with keeping it as is, but if it also works with smaller rtol, then that would be great.